### ISSUE TITLE
 [Feature Request] Support Algolia

### ISSUE THREAD

**Issue by azzamsa** [Keats/gutenberg#1745]
Hi.

I saw several project weight other SSG more just because they have [Algolia](https://www.algolia.com/doc/guides/getting-started/how-algolia-works/) support.

Would you like to support Algolia for this awesome SSG?

Related:
- https://github.com/getzola/zola/issues/1349
- https://github.com/codewars/docs/issues/248

**Comment by Keats** [comment]
Yep, I would take a PR that output the search index in the Algolia format instead of elasticlunr

**Comment by Keats** [comment]
FYI we've added support for formats for the search engine output so it's just a matter of someone adding it.
